Sources/UDPBroadcastExample/UDPBroadcastExample.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -2,6 +2,6 @@
 <Workspace
    version = "1.0">
    <FileRef
-      location = "self:UDPBroadcast.xcodeproj">
+      location = "self:UDPBroadcastExample.xcodeproj">
```

Comment:
It looks like the UDPBroadcastExample is in two workspaces? Should one be the UDPBroadcast project?